packages/zowe-explorer/__tests__/__unit__/uss/actions.unit.test.ts

```diff
@@ -25,7 +25,7 @@ import {
     createTextDocument,
     createFileResponse,
 } from "../../../__mocks__/mockCreators/shared";
-import { ValidProfileEnum } from "@zowe/zowe-explorer-api";
+import { ValidProfileEnum, ZosmfUssApi } from "@zowe/zowe-explorer-api";
```

Comment:
I'm a little confused here, where is `ZosmfUssApi` used in these tests?

Reply:
sorry was testing something while fixing test and forgot to remove it